src/module/__init__.py

```
@@ -1,6 +1,7 @@
 from .atc import register_module
 from .cf import register_module
 from .color_rand import register_module
+from .contest_manual import register_module
```

Ruff (0.8.2)

Redefinition of unused `register_module` from line 3

(F811)